great-expectations/great_expectations
# [MAINTENANCE] Enable Spark-S3 integration tests for Connecting To Your Data

- pushing my changes
- just the pipeline
- remove dependency
- remove other runme tests
- try to install spark
- yes
- let's go try this directory manipulation
- enable spark s3
- enable spark in the actual pytest run
- batch_request_options are no longer a thing
- adding aws_spark stage to docs integration
- wrong tab
- pushing to test
- Revert "[FEATURE] Enable passing "spark_config" through to "SparkDFExecutionEngine" constructor as arguments to "add_spark*()" Fluent Datasources methods. (#7810)"
- remove RUNME
- get rid of glue first
- don't over write the things in the script
- push the tests first
- just the spark version
- change to moving thing
- gimme my creds yo
- see if we can get away with ont doing the spark install
- simplify the installation
- Revert "Revert "[FEATURE] Enable passing "spark_config" through to "SparkDFExecutionEngine" constructor as arguments to "add_spark*()" Fluent Datasources methods. (#7810)""
- clean up docs-integration pipeline
- pushing the RUNME removals
- remove the sparktest yaml
- clean up so we have the minumum amount of changes
- try with uncommented
- oops
- re-enable s3 and disable aws_glue
- update the message
- remove stray RUNME
- Update ci/azure-pipelines-docs-integration.yml
- testing first batch

Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], or [CONTRIB]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:
-
-
-


After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in GitHub issues or Slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-


### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [ ] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.


Thank you for submitting!
